### PR TITLE
feat(console): events

### LINKED ISSUE #4907: Event Viewer in Console
As an admin I want to be able to access all events in the ZITADEL Console, to see the change track of my resources.

**Acceptance Criterias**
- [ ] "Log Viewer" in ZITADEL to query the events with filter criterias
- [ ] Filter possibility
       - EditorUser (FirstName, LastName, Username, ID)
       - EditorService (Name)
       - EventType 
       - Aggregate ID
       - Aggregate Type
       - Resource Owner
       - Sequence
       - CreationDate
- [ ] EventData to Show
       - EditorUser (fabienne)
       - EditorService (mgmt)
       - EventType (username.changed)
       - Aggregate ID (12345677889)
       - Aggregate Type (user)
       - Instance ID 
       - Resource Owner
       - Sequence (250)
       - CreationDate (timestamp)
       - Data (json: "username":"fabienne")

### PR DESCRIPTION
**Pull request by peintnermax**
Closes #4907 

Shows the list of events on instance level

Bildschirm­foto 2023-01-26 um 15 09 19
Bildschirm­foto 2023-01-26 um 15 10 04

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [x] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: console/src/app/pages/events/events.component.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          </ng-container>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="aggregate">
+        <th mat-header-cell *matHeaderCellDef>{{ 'IAM.EVENTS.AGGREGATE' | translate }}</th>
+        <td mat-cell *matCellDef="let event">
+          <ng-container *ngIf="event | toobject as event">
+            <div class="aggregate-row">
+              <span class="id">{{ event.aggregate.resourceOwner }}</span
```
Comment (livio-a) [inline console/src/app/pages/events/events.component.html:55]: shouldn't this be the aggregate id?

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -750,6 +752,39 @@
       "DELETE": "Entfernen",
       "DELETESUCCESS": "Gescheiterte Events entfernt."
     },
+    "EVENTS": {
+      "TITLE": "Events",
+      "DESCRIPTION": "Diese Ansicht zeigt alle eingetroffenen Events.",
```
Comment (livio-a) [inline console/src/assets/i18n/de.json:757]: eingetroffenen: aufgetretenen / vorhandenen ?

File: console/src/app/pages/events/events.component.html (hunk 1 above)
Comment (livio-a) [reply console/src/app/pages/events/events.component.html:55]: i guess we should display the resourceOwner in the dialog though

File: console/src/app/pages/events/events.component.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+              [avatarUrl]="editor.avatarUrl || ''"
+              [forColor]="editor.preferredLoginName ?? editor.displayName"
+              [size]="32"
+            >
+            </cnsl-avatar>
+            <ng-template #cog>
+              <cnsl-avatar [forColor]="editor?.preferredLoginName ?? 'franz'" [isMachine]="true">
+                <i class="las la-robot"></i>
+              </cnsl-avatar> </ng-template
+            > -->
```
Comment (livio-a) [inline console/src/app/pages/events/events.component.html:41]: i guess you want to add this as soon as the backend returns it, can you please create a user story for that and link it here?

File: console/src/app/modules/display-json-dialog/display-json-dialog.component.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        event.aggregate.type.localized.localizedMessage
+      "
+    >
+      <span class="label cnsl-secondary-text">{{ 'IAM.EVENTS.AGGREGATETYPE' | translate }}</span>
+      <span class="state aggregate-type">{{ event.aggregate.type.localized.localizedMessage }}</span>
+    </div>
+
+    <div class="data-row" *ngIf="event && event.editor?.displayName">
+      <span class="label cnsl-secondary-text">{{ 'IAM.EVENTS.EDITOR' | translate }}</span>
+      <span>{{ event.editor?.displayName }}</span>
```
Comment (livio-a) [inline console/src/app/modules/display-json-dialog/display-json-dialog.component.html:35]: we should display the id as well

**Review by livio-a — CHANGES_REQUESTED** [review]
as you already noticed, we need to be able to filter by Resource Owner, Sequence, Creation Date
and the filtering of the event types is hard as you have to scroll through the list
i'd love to have a filter for the list itself like in the proposal
image

File: console/src/app/pages/events/events.component.html (hunk 3 above)
Comment (peintnermax) [reply console/src/app/pages/events/events.component.html:41]: done https://github.com/zitadel/zitadel/issues/5136

File: console/src/app/modules/display-json-dialog/display-json-dialog.component.html (hunk 4 above)
Comment (peintnermax) [reply console/src/app/modules/display-json-dialog/display-json-dialog.component.html:35]: done

File: console/src/app/pages/events/events.component.html (hunk 1 above)
Comment (peintnermax) [reply console/src/app/pages/events/events.component.html]: yes.. fixed 👍

**Review by livio-a — APPROVED** [review]

**Pull request merged**